.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: ruby
-cache: bundler
-dist: precise
+dist: xenial
```

Comment:
do we need this ? ... can this be another PR ?

Reply:
I took it back out, one thing at a time. :)